integration/tests/economy/test_economics.py

```diff
@@ -1150,32 +1293,38 @@ def test_multiple_scheduled_trx(
         self,
         operator,
         web3_client_sol,
-        neon_user,
         increase_storage_contract,
         evm_loader,
         treasury_pool,
         sol_price,
         sol_client,
         is_dependent,
+        neon_user_with_all_tokens,
```

Comment:
why did you changed it? neon user with sols inside neon and without - 2 different cases